exercises/resistor-color-duo/resistor-color-duo.example.ts

```diff
@@ -0,0 +1,22 @@
+
+
+type colors = 'black'
+|'brown'
+|'red'
+|'orange'
+|'yellow'
+|'green'
+|'blue'
+|'violet'
+|'grey'
+|'white'
+
+const COLORS = [
+  'black', 'brown', 'red', 'orange', 'yellow', 'green',
+  'blue', 'violet', 'grey', 'white',
+]
+
+export const value = ([tens, ones, ..._]: colors[]): number => 
```

Comment:
Can you name this differently? It’s acting like a function so let’s give it a name that describes what it does.

Reply:
The [canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/resistor-color-duo/canonical-data.json) defined the property to be "value" which if I understand correctly needs to align with the method name. The same goes with your other comments, the test scenarios as well as the inputs and outputs of the methods are dictated by the `canonical-data.json`. 

Although, you do raise an interesting perspective, at what point does one deem that we need to change/update the canonical-data file. I'd assume that if that file changes, we would need to update all tracks with the implemented example/test to realign them.

Reply:
You're second paragraph is correct. This will therefore stay `value` .

Many other TypeScript track exercises embed this in a class. Perhaps that would help:

 ResistorColor that takes the colours on the constructor and gives the numerical value on a function called value (or property called value)

Reply:
sounds good